pkg/new-ui/v1beta1/frontend/src/app/models/experiment.model.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * UI relative types
+ */
+interface ExperimentCondition {
+  type: string;
+  status: string;
+  reason: string;
+  message: string;
+  lastUpdateTime: string;
+  lastTransitionTime: string;
+}
+
+export interface TrialParameterAssignments {
+  name: string;
+  value: string;
+}
+
+export interface TrialObservationMetrics {
+  name: string;
+  min: number;
+  max: number;
+  latest: number;
+}
+
+interface ExperimentCurrentOptimalTrial {
+  bestTrialName: string;
+  parameterAssignments: TrialParameterAssignments[];
+  observation: {
+    metrics: TrialObservationMetrics[];
+  };
+}
+
+export interface Experiment {
+  Name: string;
+  Namespace: string;
+  Status: string;
```

Comment:
```suggestion
  name: string;
  namespace: string;
  status: string;
```

Reply:
So right now the backend returns the `Name`, `Namespace`, `Status` and `Type` fields [ starting with a capital letter ] and also every field of the `.status` field of CR [and that's why the other fields start with a small letter]

In order to modify this in the frontend we will also need to make a small change to the backend and make these fields lowercase https://github.com/kubeflow/katib/pull/1427/files#diff-de3141fdf650f9010304f5fc6a77dda885d38e45e5590b5982a626f6f0fdc859R39-R42

Reply:
@kimwnasptd Got it.
I think it makes sense to modify the backend with the JSON annotation.
```go
type ExperimentView struct {
	Name      string `json:"name"`
	Namespace string `json:"namespace"`
	Type      string `json:"type"`
	Status    string `json:"status"`
	v1beta1experiment.ExperimentStatus
}
```

Reply:
This is really nice! I'll use 
1. use the json annotation in the backend
2. use lowercase fields in the UI

Reply:
pushed another commit that resolves this